contrib/ros/robots/robot.py

```diff
@@ -0,0 +1,33 @@
+"""
+robot Interface
+"""
+
+
+class Robot(object):
+    def move_to_start_position(self):
+        """
+        User uses this to make the robot move to a start position at the beginning of every training episode.
+        """
+        raise NotImplementedError
+
+    def set_command(self, commands):
+        """
+        User uses this to set commands to robot.
+        :param commands: object
+                    based on action_space
+        """
+        raise NotImplementedError
+
+    @property
+    def action_space(self):
+        """
+        User uses this to get robot's action_space.
+        """
+        raise NotImplementedError
+
+    def get_obs(self):
```

Comment:
Please don't abbreviate function names in public interfaces. This should be `get_observation()`.

Your API documentation should also specify the time properties of `get_observation()`. Does it return the most recent observation? Or maybe it instead returns all observations in sequence rather than the most recent one? Or maybe it guarantees that the return value of `get_observation()` is some fixed time delay after the last `send_command()`?